### DISCUSSION TITLE
Spellcheck in comments?

### DISCUSSION THREAD

**Discussion by will** [pwntester/octo.nvim#351] [Q&A]
I'm a terrible speller. Is it possible to turn on spellcheck when adding a PR comment?

**Comment by pwntester** [comment]
It should be possible, Octo buffers are regular buffers so just enable spellcheck. Is it not working for you?

**Reply by will** [reply]
It wasn't at the time for some reason. But to be honest I've been scared off of using this for a while bit though since I did a PR review where my I made 3 comments, but each got sent like 20-30 times and it made a mess :/

**Reply by pwntester** [reply]
oh wow, Im sorry about that

**Reply by will** [reply]
It's not a big problem, just strange. I should have reported it probably, but I didn't think I had enough information to make it reproducible :/
